app/controllers/admin/themes_controller.rb

```diff
@@ -299,6 +301,10 @@ def ban_in_allowlist_mode!
     raise Discourse::InvalidAccess if !GlobalSetting.allowed_theme_ids.nil?
   end
 
+  def ban_for_remote_theme!
+    raise Discourse::InvalidAccess if @theme.remote_theme
```

Comment:
Can you check if this works when using theme CLI. IIRC, themes installed via zip files are given a `remote_theme` object to store metadata, even though they aren't truely 'remote'. 

I think `theme.remote_theme&.is_git?` might be a safer check (and, we'll need to do the same for the handlebars `if` statements as well)

Reply:
After testing this one, theme cli uses local themes. Creating a new theme with CLI uploads as local, as well as installing a remote theme, downloading with cli, then uploading to the same theme id and watching. We should be OK here as is.